### ISSUE TITLE
Unable to change tab order in 5.1. Also Dashboard tabs order randomly changed after upgrading to 5.1 (5.0 -> 5.1) Possible related?

### ISSUE THREAD

**Issue by drewmiranda-gl** [Graylog2/graylog2-server#15528]
After upgrading to Graylog 5.1 (Previously on 5.0.7), i noticed some of my dashboards had their tab order randomly changed.

I can also no longer change the tab order of a dashboard. Editing and saving the tab order has no effect.

## Expected Behavior

Tab order on dashboards is never changed automatically

## Current Behavior

Tab order is randomly changed

## Steps to Reproduce (for bugs)

1. Start with Graylog 5.0.7
2. Create a dashboard with multiple tabs. Make note of the order
3. Upgrade to Graylog 5.1
4. Observe that tab order may have changed

## Context

This can cause confusion when expecting to be greeted with a specific tab when opening a dashboard or even when trying to find a specific tab.

## Your Environment

* Graylog Version: Graylog 5.1.0+14ba491
* Java Version: Bundled
* ~~Elasticsearch~~ OpenSearch Version: 2.6.0
* MongoDB Version: 6.0.6
* Operating System: Ubuntu Server 22.04 LTS
* Browser version: Google Chrome Version 113.0.5672.126 (Official Build) (arm64)

Please let me know if there are any questions or if there is any additional data i can provide.

**Comment by drewmiranda-gl** [comment]
New development, i'm not entirely unable to change the tab order 😢

**Comment by climvalencia** [comment]
Same here. Moving from 5.0.7 to 5.1.0 messed up the tab order on the dashboards and I am unable to reorder them.

**Comment by AleksJev** [comment]
Confirming random tab order issue after update from 5.0.7 to 5.1.0. 
Also tested a fresh install in docker 5.1. There is no way to change the tab order. Checked also on 5.1.0-rc.2-1 - Same problem!!!
